src/ai_client.c

```diff
@@ -3,6 +3,8 @@
  *
  * Copyright (C) 2009-2011 Keldon Jones
  *
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
Also add expansion abbreviation to line 268. (Add "AA" as it seems to be missed as well.)

An old client connecting to a new server will potentially segfault on line 515 if it receives a too new expansion. We could probably live with that (not a big use case), but for future proofing, I propose adding "?" to the list of abbreviations, and use that if the expansion from the server is too high.